frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -99,22 +101,46 @@ type State = {
   groupToDelete: ?number,
 };
 
-export async function importNmls(files: Array<File>, createGroupForEachFile: boolean) {
+export async function importTracingFiles(files: Array<File>, createGroupForEachFile: boolean) {
   try {
     const { successes: importActionsWithDatasetNames, errors } = await Utils.promiseAllWithErrors(
       files.map(async file => {
-        const nmlString = await readFileAsText(file);
+        const ext = _.last(file.name.split("."));
         try {
-          const { trees, treeGroups, datasetName } = await parseNml(
-            nmlString,
-            createGroupForEachFile ? file.name : null,
-          );
-          return {
-            importAction: addTreesAndGroupsAction(trees, treeGroups),
-            datasetName,
-          };
+          if (ext === "nml") {
+            const nmlString = await readFileAsText(file);
+            const { trees, treeGroups, datasetName } = await parseNml(
+              nmlString,
+              createGroupForEachFile ? file.name : null,
```

Comment:
The `createGroupForEachFile` setting won't work for protobufs now. I had a quick look and there is a `wrapInNewGroup` method in the `nml_helpers.js` which is used for that. I think it should be straightforward to extract that wrapping behavior and use it in this method for both NMLs and Protobufs :)

Reply:
Ah, nice, thanks for catching this! I just fixed it!